components/post_view/reaction/reaction.jsx

```diff
@@ -246,9 +249,10 @@ export default class Reaction extends React.PureComponent {
                 }
                 onEnter={this.loadMissingProfiles}
             >
-                <div
+                <button
                     id={`postReaction-${this.props.post.id}-${this.props.emojiName}`}
-                    className={className}
+                    aria-label={ariaLabelEmoji}
+                    className={'style--none ' + className}
```

Comment:
Maybe switch to template literals as you have done elsewhere, like:
```suggestion
                    className={`style--none ${className}`}
```